src/move/econia/sources/core.move

```diff
@@ -16,22 +16,37 @@ module econia::core {
     use aptos_framework::aptos_coin;
     #[test_only]
     use econia::test_assets;
+    #[test_only]
+    use std::features;
+    #[test_only]
+    use econia::rational;
 
-    const GENESIS_UTILITY_ASSET_METADATA_ADDRESS: address = @aptos_framework;
+    const GENESIS_UTILITY_ASSET_METADATA_ADDRESS: address = @aptos_fungible_asset;
     const GENESIS_MARKET_REGISTRATION_FEE: u64 = 100_000_000;
     const GENESIS_ORACLE_FEE: u64 = 0;
     const GENESIS_INTEGRATOR_WITHDRAWAL_FEE: u64 = 0;
+    const GENESIS_BOOK_MAP_INNER_NODE_ORDER: u16 = 7;
+    const GENESIS_BOOK_MAP_LEAF_NODE_ORDER: u16 = 10;
+    const GENESIS_TICK_MAP_INNER_NODE_ORDER: u16 = 20;
+    const GENESIS_TICK_MAP_LEAF_NODE_ORDER: u16 = 12;
 
-    const GENESIS_DEFAULT_POOL_FEE_RATE_BPS: u8 = 30;
-    const GENESIS_DEFAULT_TAKER_FEE_RATE_BPS: u8 = 0;
+    const GENESIS_DEFAULT_POOL_FEE_RATE: u16 = 3_000;
+    const GENESIS_DEFAULT_PROTOCOL_FEE_RATE: u16 = 0;
     const GENESIS_DEFAULT_MAX_PRICE_SIG_FIGS: u8 = 4;
     const GENESIS_DEFAULT_EVICTION_TREE_HEIGHT: u8 = 5;
-    const GENESIS_DEFAULT_EVICTION_PRICE_DIVISOR_ASK: u128 = 100_000_000_000_000_000_000;
-    const GENESIS_DEFAULT_EVICTION_PRICE_DIVISOR_BID: u128 = 100_000_000_000_000_000_000;
-    const GENESIS_DEFAULT_EVICTION_LIQUIDITY_DIVISOR: u128 = 1_000_000_000_000_000_000_000_000;
+    const GENESIS_DEFAULT_EVICTION_PRICE_RATIO_ASK_NUMERATOR: u64 = 1;
+    const GENESIS_DEFAULT_EVICTION_PRICE_RATIO_ASK_DENOMINATOR: u64 = 3;
+    const GENESIS_DEFAULT_EVICTION_PRICE_RATIO_BID_NUMERATOR: u64 = 1;
+    const GENESIS_DEFAULT_EVICTION_PRICE_RATIO_BID_DENOMINATOR: u64 = 5;
+    const GENESIS_DEFAULT_EVICTION_LIQUIDITY_RATIO_NUMERATOR: u64 = 1;
+    const GENESIS_DEFAULT_EVICTION_LIQUIDITY_RATIO_DENOMINATOR: u64 = 100_000;
     const GENESIS_DEFAULT_INNER_NODE_ORDER: u8 = 10;
     const GENESIS_DEFAULT_LEAF_NODE_ORDER: u8 = 5;
 
+    const COMPARE_LEFT_GREATER: u8 = 0;
```

Comment:
Any reason you don't use `rational::get_COMPARE_LEFT_GREATER` instead of redefining the constant here and then asserting that they're equal? Not a big deal, obviously, but seems a bit roundabout